kwanghwi/myapp/app.js

```diff
@@ -37,23 +37,135 @@ app.get("/ping", (req, res) => {
 });
 
 app.post("/user/signup", async (req, res) => {
-  const { userId, password, email, profileImage } = req.body;
+  const { userId, password, name, email, profileImage } = req.body;
   await appDataSource.query(
     `INSERT INTO users(
-      user_id,
+      userId,
       password,
+      name,
       email,
       profileImage
-    ) VALUES (?, ?, ?, ?);
+    ) VALUES (?, ?, ?, ?, ?);
     `,
-    [userId, password, email, profileImage]
+    [userId, password, name, email, profileImage]
   );
   return res.status(201).json({ message: "signup success!" });
 });
 
+app.post("/post/create", async (req, res) => {
```

Comment:
```suggestion
app.post("/posts", async (req, res) => {
```
우리는 이미 http method를 통해 post(등록) 한다는 메세지를 잘 전달하고 있습니다. 
이처럼 무엇을 나타내는 what(게시글)과, 어떻게를 나타내는 how(등록)의 두가지를 엔드포인트와 http method를 통해 간단하고 명확하게 나타낼 수 있습니다. 마치 아래의 경우와 같이요!
```shell
http -v GET localhost:3000/users/1          # db의 유저들중 1번(/users/1)에 해당하는 유저의 정보의 가져와(GET)줘
``` 

이러한 방법으로 api endpoint를 작성하는 것을 RESTful한 api를 작성한다고 일컫고 있습니다. 이는 차주에 있을 Building RESTful APIs 세션에서 더 쉽고 재미있게 다룰 예정이니 참고해주세요-!